### PR TITLE
Trim whitespace from returned secrets

### PR DESCRIPTION
**Pull request by kargakis**
Fixes https://github.com/kubernetes/test-infra/issues/8877

### CODE REVIEW

File: prow/config/config.go [hunk 1]
```diff
@@ -392,7 +393,7 @@ func LoadSingleSecret(path string) ([]byte, error) {
 	if err != nil {
 		return nil, fmt.Errorf("error reading %s: %v", path, err)
 	}
-	return b, nil
+	return bytes.TrimSpace(b), nil
```
Comment (BenTheElder) [inline prow/config/config.go]: Doesn't this just mean the secret is incorrect? /shrug

Comment (kargakis) [reply prow/config/config.go]: Yeah, though this is what we used to do before #8877 anyhow :/

Comment (kargakis) [reply prow/config/config.go]: Sorry, before https://github.com/kubernetes/test-infra/pull/8579

File: prow/config/config.go [hunk 2]
```diff
@@ -31,7 +32,7 @@ import (
 
 	"github.com/ghodss/yaml"
 	"github.com/sirupsen/logrus"
-	"gopkg.in/robfig/cron.v2"
+	cron "gopkg.in/robfig/cron.v2"
```
Comment (BenTheElder) [inline prow/config/config.go]: Saw this change in Paul's PR as well, what's up here?

Comment (kargakis) [reply prow/config/config.go]: goimports in my editor (ST3) removes this import because it's used as `cron` in the code.

**Comment by BenTheElder** [comment]
/hold cancel
If anyone has qualms here we can revisit later, merging to unbreak https://github.com/kubernetes/test-infra/issues/8877

**Comment by kargakis** [comment]
@BenTheElder thanks!

File: prow/config/config.go (hunk 2 above)
Comment (BenTheElder) [reply prow/config/config.go]: Thanks, xref-ing

**Pull request merged**